_includes/code/python/howto.configure.rbac.roles.py

```diff
@@ -0,0 +1,125 @@
+import weaviate
+
+# START AdminClient 
+from weaviate.classes.init import Auth
+
+# Connect to Weaviate as Admin
+admin_client = weaviate.connect_to_local(auth_credentials=Auth.api_key("admin-key"))
+# END AdminClient
+
+# START CreateRole
+from weaviate.classes.rbac import Permissions
+
+admin_client.roles.create(
+    role_name="devrel", permissions=<LIST-OF-PERMISSIONS>
```

Comment:
I would prefer if the code would be runnable (for internal QC reasons)

What if it was set up like this?

```python
permissions_to_add = [
   Permissions.data(collection="Testing_*", read=True, create=True)
]

# START CreateRole
from weaviate.classes.rbac import Permissions

admin_client.roles.create(
    role_name="devrel", permissions=permissions_to_add
```

This way, the user only sees the var name `permissions_to_add`, but it is still runnable at our end